src/App.tsx

```diff
@@ -20,8 +20,12 @@ const StyledApp = styled.div`
 const config: Config = {
   autoConnect: false,
   readOnlyUrls: {
-    [Mainnet.chainId]: getDefaultProvider("mainnet"),
-    [Rinkeby.chainId]: getDefaultProvider("rinkeby"),
+    [Mainnet.chainId]: new ethers.providers.JsonRpcProvider(
+      "https://eth-mainnet.g.alchemy.com/v2/8iVRJ1Rswjzp0S3ySwiWOaS0jtoZ4uRg"
```

Comment:
If we haven't already, we should go into the Alchemy settings and make sure that this key is restricted to only work when coming from the `aikovirtual.com` domain. Otherwise users can take this key and use it themselves and use up all our credit.